includes/admin/reporting/reports.php

```diff
@@ -2881,7 +2881,7 @@ function display_export_report() {
 				</div>
 
 				<div class="postbox edd-export-downloads">
-					<h2 class="hndle"><span><?php esc_html_e( sprintf( __( 'Export %s', 'easy-digital-downloads' ), edd_get_label_plural() ) ); ?></span></h2>
+					<h2 class="hndle"><span><?php esc_html_e( sprintf( __( 'Export %s Products', 'easy-digital-downloads' ), edd_get_label_singular() ) ); ?></span></h2>
```

Comment:
While we are here, could we go ahead and add a translators note for the placeholder?

I also feel like it would be nice to change the wording on line 2886 as it ends up reading "Download a CSV of downloads."